Robus/src/robus.c

```diff
@@ -419,6 +419,11 @@ static error_return_t Robus_MsgHandler(msg_t *input)
             Robus_SetNodeDetected(NETWORK_LINK_CONNECT);
             return SUCCEED;
             break;
+        case END_DETECTION:
+            // detect end of detection
+            Robus_SetNodeDetected(NETWORK_LINK_UP);
+            return SUCCEED;
```

Comment:
I think sometimes users will need to get the END_DETECTION event allowing them to configure things.
By making it this way on Robus they have to find back the detection event by constantly polling if NETWORK_LINK_DOWN then NETWORK_LINK_UP.
We could make this message go to users allowing them to deal with this event.
Also I still convinced that this kind of msg management should be managed on Luos.

Reply:
@ncros3  This is a great idea, it will simplify the code into the services that launch a detection. I can occupy myself with it if you want

Reply:
@nicolas-rabault However if we implement this method we need to well inform the user what this end detection message is, and I am afraid that it will add a bit of compexity in what the user needs to know from the interior of Luos process in order to make their network properly work...

Reply:
✅ I have done this. Now END_DETECTION arrives to the services callback. HOWEVER, WE NEED TO MAKE REALLY CLEAR TO THE USERS THAT THEY NEED A MSG_HANDLER. Because it is possible that they dont pull this message and it stays stored . TO SPECIFY IN DOC

Reply:
I think this discussion goes out of the scope of this PR. I agree with the fact that the user shoudn't have to deal with the network connection state in the service code, luos should do it for him. 

One way to do that is to forbid access to the network if the node is not connected: the user can call "Luos_sendMsg" from a service and luos check if the message can be send to the network. If not (because we're not in NETWORK_LINK_UP" state), luos don't send the message and can return an error code to the user. 

We can discuss further about this feature and its implementation but I think it's not the point of this PR.

Reply:
I think @BiliouriV already included it in this PR.
@BiliouriV if yes please resolve this conversation.